src/tools/cephfs/cephfs-shell

```diff
@@ -535,9 +533,8 @@ exists.')
                 try:
                     cephfs.mkdir(path, permission)
                 except libcephfs.Error:
-                    self.perror('directory missing in the path; '
```

Comment:
Before libcephfs.Error, add exception to catch `ObjectNotFound` error. 
For this exception let the error message be `directory missing in the path; you may want to pass the -p argument`.

Reply:
Right now it prints `error in mkdir y/z: No such file or directory` which IMO is good enough. coreutils `mkdir` prints similar message -
```
$ mkdir y/z
mkdir: cannot create directory ‘y/z’: No such file or directory
```

Reply:
Here we want the user to know  about the `-p` option. For other errors we can have standard messages.